src/main/java/com/autotune/common/k8sObjects/ExperimentResultData.java

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.common.k8sObjects;
+
+import com.autotune.utils.AnalyzerConstants;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Experiment result storage object.
+ */
+public class ExperimentResultData {
```

Comment:
In general we need to have classes inside `k8sObjects` dir if it's associated with a CRD or is directly related to k8s

Reply:
Agreed moved to different directory.